lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -45,7 +50,15 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
+  <span ng-show="modal.state.getDrive().system" class="modal-footer-warning">
+      Etcher has identified this as a system drive
+  </span>
   <button class="button button-primary button-block"
+    ng-class="{ 'label-danger': modal.state.getDrive().system, 'button-primary': !modal.state.getDrive().system }"
```

Comment:
`button-primary` is both in `ngClass` and in the element's `class` attribute. I believe your intention was to but the "danger" class in `ngClass`, such that is gets added when the drive is a system drive, right?

(BTW, the button doesn't go red here for system drives)

![screenshot 2016-12-11 20 32 50](https://cloud.githubusercontent.com/assets/2192773/21084941/00f20c54-bfe1-11e6-835d-82a65981ab3a.png)